content/source/docs/enterprise-beta/api/modules.html.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: enterprise2
+page_title: "Modules - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-modules"
+---
+
+# Registry Modules API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## Listing and reading modules, providers and versions
+
+The Terraform Enterprise Module Registry provides the same API endpoints as the [Terraform Registry HTTP API](../../registry/api.html). Terraform Enterprise extends this API to add [authentication](./index.html#authentication). Refer to the [Module Registry HTTP API](../../registry/api.html) to perform the following:
+
+- List available versions for a specific module
+- Download source code for a specific module version
+- List latest version of a module for all providers
+- Get the latest version for a specific module provider
+- Get a specific module
+- Download the latest version of a module
+
+The TFE Module Registry endpoints differs from the Module Registry endpoints in the following ways:
+
+- The `:namespace` parameter should be replaced with the organization name.
+- All other TFE endpoints are on the `/api/v2` subpath; the module registry endpoint has the subpath `/api/registry`.
```

Comment:
It's `/api/v1/modules`. I suggest changing this to:

> The module registry endpoints have the path prefix provided in the [discovery document](../../registry/api.html#service-discovery) which is currently`/api/v1/modules`.

Reply:
The discovery endpoint is `/api/registry/v1` the CUD endpoints are on `/api/v2/registry-modules`; or are you saying that this is changing?